conseil-common/src/main/scala/tech/cryptonomic/conseil/common/rpc/RpcClient.scala

```diff
@@ -0,0 +1,75 @@
+package tech.cryptonomic.conseil.common.rpc
+
+import scala.util.control.NoStackTrace
+
+import cats.effect.Concurrent
+import com.typesafe.scalalogging.LazyLogging
+import fs2.Stream
+import org.http4s.client._
+import org.http4s.client.dsl.Http4sClientDsl
+import org.http4s.{EntityDecoder, EntityEncoder, Method, Uri, Header}
+
+import tech.cryptonomic.conseil.common.rpc.RpcClient._
+
+class RpcClient[F[_]: Concurrent](
```

Comment:
Would you kindly provide documentation, both for usage and, possibly, for design notes?
It would be appreciated.

Reply:
It's an implementation of https://www.jsonrpc.org/specification client.

> The Tezos node provides a JSON/RPC interface. Note that it is an RPC, and it is JSON based, but it does not follow the “JSON-RPC” protocol. 

So there is a difference, actually, Tezos exposes rest json API not related to JSON-RPC protocol. So the naming can be misleading.

Reply:
good, then we can put that in the doc comments, with reference to the spec.